pkgs/project-zot/zot/scaffold.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/aqua-generate-registry.json
+# aqua - Declarative CLI Version Manager
+# https://aquaproj.github.io/
+# Other than name is optional. All initial values are just examples.
+name: project-zot/zot
+version_filter: not (Version matches "-rc[0-9]+$")
+# version_prefix: cli-
+all_assets_filter: ((Asset matches "^zot-") and (not (Asset matches "-(debug|minimal)")))
```

Comment:
Unfortunately, this excludes `checksums.sha256.txt`.
We're aware that we should improve the feature to filter assets.

Reply:
Fixed to include the checksum file

Reply:
After checking past release files, it seems like it could be written even better.
I will make some further revisions.

Reply:
Since the old version of the assets cannot be detected, we will make further improvements.